ethpm_types/contract_type.py

```diff
@@ -220,7 +220,9 @@ def __init__(
         self._selector_hash_fn = selector_hash_fn
         super().__init__(iterable)
 
-    def __getitem__(self, item: Union[int, slice, str, bytes, MethodABI, EventABI]):  # type: ignore
+    def __getitem__(  # type: ignore
+        self, item: Union[int, slice, str, HexBytes, bytes, MethodABI, EventABI]
```

Comment:
Curious, have we looked into making `HexBytes` a subclass of `bytes` ?

Reply:
It should already be

Reply:
Should we remove `HexBytes` from the type list here since it is covered by bytes? Or is bytes not needed anymore?